.github/codeql/codeql-config-tests.yml

```diff
@@ -0,0 +1,7 @@
+paths:
+  - packages/**/__tests__/**
+  - packages/**/*.test.*
+  - packages/**/__e2e__/**
+query-filters:
+  - exclude:
+      id: js/insecure-temporary-file
```

Comment:
This keeps being flagged in main codebase.
![image](https://user-images.githubusercontent.com/5849952/221246671-b1fe75d2-4529-41df-bae9-efc32d3cb92c.png)